derived_images/mongodb/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM opensuse:42.1
```

Comment:
I think that the proper base image is `FROM opensuse/amd64:42.1` now. At least that's what we use for the Portus image :wink:.